robottelo/ui/locators.py

```diff
@@ -1040,7 +1042,20 @@ def __iter__(self):
                    " and contains(.,'%s')]")),
     "puppetclass.delete": (
         By.XPATH, "//a[@class='delete' and contains(@data-confirm, '%s')]"),
-
+    "puppetclass.import": (
+        By.XPATH, "//a[contains(@href,'import')]"),
+    "puppetclass.environment_default_check": (
+        By.XPATH, "//input[contains(@id,'KT_Default_Organization_Library')]"),
+    "puppetclass.update": (
+        By.XPATH, "//input[@value='Update']"),
+    "puppetclass.paramfilter": (
+        By.XPATH, "//input[contains(@placeholder,'Filter')]"),
+    "puppetclass.parameter": (
+        By.XPATH, "//li[@class='active search-marker']"),
+    "puppetclass.paramDescription": (
```

Comment:
Several referenes to `paramDescription` are made in the test. I assume those will need to change too.